src/components/MonthYear/MonthYear.stories.jsx

```diff
@@ -6,9 +6,9 @@ export default {
   component: MonthYear,
 };
 
-const Template = args => {
+const Template = (args) => {
   const [date, setDate] = useState(args.date);
-  const onChange = newDate => {
+  const onChange = (newDate) => {
```

Comment:
I accidentally committed this chunk, so I _also_ made the `.prettierrc` change in here to prevent this from happening again in the future.